test/helpers/foundry/extract.ts

```diff
@@ -68,16 +68,32 @@ export async function extractFrom(
       paths.push(join(dir, relative(tempDir, path)));
     }
 
+    // this directory shouldn't exist, but if two simultaneous `yarn foundryup`
+    // processes are running, it might. Last process wins, so we remove other
+    // `dir`s just in case.
+    await rm(dir, rmOpts);
```

Comment:
This change might prevent `rename` issues, even though it should be impossible to get into this function is `dir` already exists.